pkg/controller/perconaservermongodb/psmdb_controller.go

```diff
@@ -446,6 +447,20 @@ func (r *ReconcilePerconaServerMongoDB) reconcileStatefulSet(arbiter bool, cr *a
 			)
 		}
 	}
+
+	sslHash, err := r.getTLSHash(cr, cr.Spec.Secrets.SSL)
+	if err != nil {
+		return nil, fmt.Errorf("get secret hash error: %v", err)
+	}
+	sfsSpec.Template.Annotations["percona.com/ssl-hash"] = sslHash
+	sslInternalHash, err := r.getTLSHash(cr, cr.Spec.Secrets.SSLInternal)
+	if err != nil && !k8serrors.IsNotFound(err) {
+		return nil, fmt.Errorf("get secret hash error: %v", err)
+	}
+	if !k8serrors.IsNotFound(err) {
+		sfsSpec.Template.Annotations["percona.com/ssl-internal-hash"] = sslInternalHash
```

Comment:
Unreachable code. `!k8serrors.IsNotFound(err)` would be triggered in previous `if`